src/verilog/parser.y

```diff
@@ -858,13 +881,20 @@ module_item_brace:
 	;
 
 module_or_generate_item:
-	  attribute_instance_brace parameter_override { $$=$2; }
-	| attribute_instance_brace gate_instantiation { $$=$2; }
-	// | attribute_instance_brace udp_instantiation { $$=$2; }
-	| attribute_instance_brace module_instantiation { $$=$2; }
-        | attribute_instance_brace smv_using { $$ = $2; }
-        | attribute_instance_brace smv_assume { $$ = $2; }
-	| attribute_instance_brace module_common_item { $$=$2; }
+	  attribute_instance_brace parameter_override
+		{ add_attributes($2, $1); $$=$2; }
+	| attribute_instance_brace gate_instantiation
+		{ add_attributes($2, $1); $$=$2; }
+	// | attribute_instance_brace udp_instantiation
+	//	{ add_attributes($2, $1); $$=$2; }
```

Comment:
Why was this always commented out (and still is)?

Reply:
Perhaps you can document this in source via a comment?

Reply:
Done